### PR TITLE
Clear extensions cache after discover/execution complete.

### PR DESCRIPTION
**Pull request by Faizan2304**

### CODE REVIEW

**Comment by Faizan2304** [comment]
@dotnet-bot test Windows_NT / Release Build

**Comment by Faizan2304** [comment]
@dotnet-bot test Windows_NT / Release Build

File: src/Microsoft.TestPlatform.Client/Discovery/DiscoveryRequest.cs [hunk 1]
```diff
@@ -225,6 +226,7 @@ public void HandleDiscoveryComplete(long totalTests, IEnumerable<TestCase> lastC
                 }
                 finally
                 {
+                    TestPluginCache.Instance.ClearExtentions();
```
Comment (codito) [inline src/Microsoft.TestPlatform.Client/Discovery/DiscoveryRequest.cs]: nit: API spelling is incorrect.

Comment (Faizan2304) [inline src/Microsoft.TestPlatform.Client/Discovery/DiscoveryRequest.cs]: Fixed!!

**Comment by Faizan2304** [comment]
@dotnet-bot test this please

File: src/Microsoft.TestPlatform.Client/Discovery/DiscoveryRequest.cs [hunk 2]
```diff
@@ -8,6 +8,7 @@ namespace Microsoft.VisualStudio.TestPlatform.Client.Discovery
     using System.Linq;
     using System.Threading;
 
+    using Microsoft.VisualStudio.TestPlatform.Common.ExtensionFramework;
```
Comment (codito) [inline src/Microsoft.TestPlatform.Client/Discovery/DiscoveryRequest.cs]: nit: is this used in the code? There is not new addition in the file.

**Review by codito — APPROVED** [review]

**Comment by codito** [comment]
@dotnet-bot test this please

**Comment by Faizan2304** [comment]
@dotnet-bot test Windows_NT / Debug Build please

**Pull request merged**